metrics/web/lib/common/presentation/widgets/metrics_text_form_field.dart

```diff
@@ -154,7 +140,11 @@ class _MetricsTextFormFieldState extends State<MetricsTextFormField> {
             obscureText: widget.obscureText,
             keyboardType: widget.keyboardType,
             style: textFieldTheme.textStyle,
-            decoration: _decoration,
+            decoration: _decoration.copyWith(
```

Comment:
I like this! But could you please check that the prefix icon matches the design (for the search field, for example)?